src/storage/buffer/file_worker/file_worker.cpp

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 module;
+#include "base64.hpp"
```

Comment:
Do we really need this header?

Reply:
I think so

Reply:
like src/storage/fast_rough_filter/probabilistic_data_filter.cpp and src/storage/fast_rough_filter/min_max_data_filter.cpp